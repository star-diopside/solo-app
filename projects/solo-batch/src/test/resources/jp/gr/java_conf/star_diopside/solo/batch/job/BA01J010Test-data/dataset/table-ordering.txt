users
authorities
user_schedule
