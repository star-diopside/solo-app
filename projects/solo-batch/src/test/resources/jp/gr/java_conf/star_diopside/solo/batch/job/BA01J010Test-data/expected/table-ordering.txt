users
authorities
schedules
user_schedule
